### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by tayronemachadozup**
**- What I did**

Removed deprecated code of all beagle core

**- How I did it**

Removing all @deprecated code and checking for possible impacts 

**- How to verify it**

Was verified through unit tests, pocs with react and angular and E2E test

**- Description for the changelog**

### CODE REVIEW

File: __tests__/unit/old-structure/middlewares/beagleStyleClassMiddleware.spec.ts [hunk 1]
```diff
@@ -1,45 +0,0 @@
-/*
```
Comment (Tiagoperes) [inline __tests__/unit/old-structure/middlewares/beagleStyleClassMiddleware.spec.ts:1]: I don't think this file should have been removed. Despite its name, nothing here has been deprecated.

File: __tests__/unit/old-structure/middlewares/beagleStyleMiddleware.spec.ts [hunk 2]
```diff
@@ -1,142 +0,0 @@
-/*
```
Comment (Tiagoperes) [inline __tests__/unit/old-structure/middlewares/beagleStyleMiddleware.spec.ts:1]: I don't think this file should have been removed. Despite its name, nothing here has been deprecated.

File: __tests__/unit/old-structure/utils/test-utils.ts [hunk 3]
```diff
@@ -258,7 +257,7 @@ export function createBeagleViewMock(custom: PartialBeagleView = {}): BeagleView
 
 export function createPreFetcherMock(custom: Partial<PreFetcher> = {}): PreFetcher {
   return {
-    fetch: custom.fetch || jest.fn(),
```
Comment (Tiagoperes) [inline __tests__/unit/old-structure/utils/test-utils.ts:261]: I don't get it, why did you need to change this?

File: __tests__/unit/old-structure/middlewares/beagleConvertToChildren.spec.ts [hunk 4]
```diff
@@ -1,44 +0,0 @@
-/*
```
Comment (Tiagoperes) [inline __tests__/unit/old-structure/middlewares/beagleConvertToChildren.spec.ts:1]: I don't think this file should have been removed. Despite its name, nothing here has been deprecated.

**Review by Tiagoperes — CHANGES_REQUESTED** [review]

File: __tests__/unit/old-structure/middlewares/beagleStyleClassMiddleware.spec.ts (hunk 1 above)
Comment (tayronemachadozup) [reply __tests__/unit/old-structure/middlewares/beagleStyleClassMiddleware.spec.ts]: Really this file shouldn't have been removed.

File: __tests__/unit/old-structure/middlewares/beagleStyleMiddleware.spec.ts (hunk 2 above)
Comment (tayronemachadozup) [reply __tests__/unit/old-structure/middlewares/beagleStyleMiddleware.spec.ts]: Really this file shouldn't have been removed.

File: __tests__/unit/old-structure/middlewares/beagleConvertToChildren.spec.ts (hunk 4 above)
Comment (tayronemachadozup) [reply __tests__/unit/old-structure/middlewares/beagleConvertToChildren.spec.ts]: Really this file shouldn't have been removed.

File: __tests__/unit/old-structure/utils/test-utils.ts (hunk 3 above)
Comment (tayronemachadozup) [reply __tests__/unit/old-structure/utils/test-utils.ts:261]: It shouldn't have been changed

**Review by Tiagoperes — APPROVED** [review]

**Review by Tiagoperes — APPROVED** [review]

**Review by arthurbleilzup — APPROVED** [review]
